README.md

```diff
@@ -96,6 +96,9 @@ To program the FPGA, either use FuseSoC again
 
 ```
 fusesoc --cores-root=. run --target=synth --run lowrisc:ibex:demo_system
+
+# If the above does not work, try executing the programming operation manually with..
+make -C ./build/lowrisc_ibex_demo_system_0/synth-vivado/ pgm
```

Comment:
Nice, I had found this was broken for me as well. I've been programming using the Vivado GUI. Also, we should be able to use the openFPGALoader, which I have vendored in with this PR: https://github.com/lowRISC/ibex-demo-system/pull/20